src/commands/logStream/ReplicaListStep.ts

```diff
@@ -0,0 +1,37 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { ContainerAppsAPIClient, Replica } from "@azure/arm-appcontainers";
+import { AzureWizardPromptStep, IAzureQuickPickItem, createSubscriptionContext, nonNullProp, nonNullValue } from "@microsoft/vscode-azext-utils";
+import * as dayjs from 'dayjs';
+import { createContainerAppsAPIClient } from "../../utils/azureClients";
+import { localize } from "../../utils/localize";
+import { IStreamLogsContext } from "./IStreamLogsContext";
+
+export class ReplicaListStep extends AzureWizardPromptStep<IStreamLogsContext> {
+    public async prompt(context: IStreamLogsContext): Promise<void> {
+        const placeHolder: string = localize('selectReplica', 'Select a Replica');
```

Comment:
nit: lowercase replica. We typically only capitalize things that have Azure branding in it